src/control/controlbehavior.h

```diff
@@ -30,6 +30,15 @@ class ControlNumericBehavior {
             MidiOpCode o, double dParam, ControlDoublePrivate* pControl);
 };
 
+// ControlSelectBehavior passes the midi value directly to the internal parameter value.  It's
+// useful for selector knobs that pass +1 in one direction and -1 in the other.
+class ControlSelectBehavior : public ControlNumericBehavior {
+  public:
+    ControlSelectBehavior() {}
+    double midiToParameter(double midiValue) override;
+    double valueToMidiParameter(double dValue) override;
```

Comment:
We also need to override:
valueToParameter() and parameterToValue();
for value in the range of 0 ... 1

Reply:
When I tested, the existing non-overridden handlers appeared to work as-is... are you sure it's necessary?

Reply:
I also have trouble understanding what the difference is between value and parameter, so if you can explain that that might help me

Reply:
It would be necessary to bind non midi controls to the control. 

Each control value have three appearances:
value =  The business logic value with a unit, intended to be displayed to the user.
midi = The value transfered in the midi scale of 0 ... 0x7F
parameter = The normalized and linearized value 0 ... 1 

Not sure what could be the range covered by parameter in this case.